lib/plugins.js

```diff
@@ -30,6 +30,15 @@ const PONG_TTL = '1 week'
 const tradesQueues = {}
 
 function plugins (settings, deviceId) {
+  function getCommissionForTx (tx) {
```

Comment:
I think it would be better to add the commission to the tx when it's created, e.g.: https://github.com/lamassu/lamassu-machine/blob/dev/lib/brain.js#L1722

Then we could use that setting when we do the rate computations, as well.

Reply:
uhm, I'm not sure it's a good idea to let l-m send commission percentage.
Also I'm calling this function on ls postTx API which is the same as https://github.com/lamassu/lamassu-machine/blob/dev/lib/brain.js#L1722 but done on l-s side.

Reply:
I was checking on l-m, I think we only output finished rates so we would need to expose this setting through l-s API, to allow l-m set it on tx creation and then send it through postTx API.
Seems kind of complicated to send back and forth this number while we already have it on l-s side.

Reply:
Not sure. We just have to make sure we're using the same value for the actual commission and price calculations.